test/unit/version_test.rb

```diff
@@ -113,7 +113,7 @@ class VersionTest < ActiveSupport::TestCase
 
   context "updated gems" do
     setup do
-      Timecop.freeze Time.zone.today
+      travel_to Time.zone.today
```

Comment:
@simi I didn't investigate usage of Timecop one by one because I just want to convert it to activesupport helper but I think you're right, it's not needed here. Thank you so much for all review.              :yellow_heart: :blue_heart: :heart: